src/illuminatio/test_generator.py

```diff
@@ -1,3 +1,6 @@
+"""
+file for test case generation
```

Comment:
Start with a capital letter to match the rest of the docstrings (see: https://www.python.org/dev/peps/pep-0257/)